src/components/DpButton/DpButton.vue

```diff
@@ -88,6 +89,16 @@ export default {
       default: ''
     },
 
+    /**
+     * Icon that will be placed before button text.
```

Comment:
See my previous comment (sizing for both icons).

```suggestion
     * Define the size of the button icons.
```